demos/smart_classroom_demo/action_event_metrics.py

```diff
@@ -23,6 +23,11 @@
 from lxml import etree
 from tqdm import tqdm
 
+try:
+    xrange          # Python 2
+except NameError:
+    xrange = range  # Python 3
+
```

Comment:
@cclauss, I think the more elegant way to fix the compatibility issue is to add `from builtins import range` import row and replace all `xrange` onto `range`.

Reply:
Changed but without a requirements.txt or setup.py it was difficult to determine if _future_ or six were installed dependencies so I did it the old fashioned way.